Products/PloneKeywordManager/testing.py

```diff
@@ -27,6 +39,9 @@ def setUpPloneSite(self, portal):
         self.applyProfile(portal, 'Products.PloneKeywordManager:default')
         self.applyProfile(portal, 'plone.app.dexterity:default')
 
+        if PLONE_5:
```

Comment:
IMO Plone 4.x should test against Archetypes and Plone 5.x should test against plone.app.contenttypes only; that would keep the test fixture clean and sane.